cumulus-client/src/test/resources/cumulus/json/idPConfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "token_endpoint": "https://api.example.com/oauth2/token",
```

Comment:
The well known includes the token endpoint, it doesn't need to be configured separately.

Reply:
Updated to use the well known endpoint. the config includes this token_endpoint as well, but its now being ignored

see: https://github.com/USACE/cumulus/issues/444#issuecomment-2787333254

Reply:
if it's ignored shouldn't we just delete it?

Reply:
this json is intended to mimic the payload we get back from that identity-provider/config endpoint from cumulus. I think we want to include that here to be true to that payload ?

Reply:
Oh I see, that makes sense.